test/test_utils/sagemaker.py

```diff
@@ -274,7 +274,7 @@ def execute_sagemaker_remote_tests(image):
             if res.failed:
                 raise DLCSageMakerRemoteTestFailure(
                     f"{pytest_command} failed with error code: {res.return_code}\n"
-                    f"Traceback:\n{res.stdout}"
+                    f"Traceback:\n{res.stderr}"
```

Comment:
nit: instead of 'traceback', how about just dump stdout and stderr directly and label them as such?

```
                raise DLCSageMakerRemoteTestFailure(
                    f"{pytest_command} failed with error code: {res.return_code}\n"
                    f"stdout:\n{res.stdout}\n"
                    f"stderr:\n{res.stderr}")
```

Reply:
Cool.